### DISCUSSION TITLE
Menus disappearing.

### DISCUSSION THREAD

**Discussion by holden** [swellstores/community#364] [Help] [answered]
Every time I use the storefront editor today to make any kind of change, my header and footers disappear.  I'm using the origin theme.

I can add them back if I restore the default settings but as soon as I make any change using the storefront editor, eg changing the logo, change any product settings, anything, even if I leave it open and view a page it immediately removes my menus from the theme. They are still there in navigation, I can see them, they just disappear from the website without any noticeable errors in the console.

I tried deleting the storefront completely and re-adding it and all the menus but it does the same thing as soon as I touch the storefront editor all the headers & footers vanish. 

There must be something wrong in my settings?

The only thing I was doing today that might have caused this, is I modified the `heading_with_text.json` to change the `description` field from `basic_html` to `rich_html` as I wanted to be able and add some arbitrary HTML into a field.  I don't know if this was a coincidence or the cause but it's the only thing I can think of that I was doing at the time.

Running this temporarily fixes things:
```
executeCommand([
  'swell storefronts push',
  '--id $SWELL_STOREFRONT_ID',
  '-t editor',
  '-f ./config/editor.json',
  '--secret-key $SWELL_SECRET_KEY',
  '--store $SWELL_STORE_ID',
]);

executeCommand([
  'swell storefronts push',
  '--id $SWELL_STOREFRONT_ID',
  '-t settings',
  '-f ./config/defaults.json',
  '--secret-key $SWELL_SECRET_KEY',
  '--store $SWELL_STORE_ID',
]);
```

How can I even debug this?

**Comment by logeshswell** [comment]
Can you help me with the STORE_URL?

**Reply by holden** [reply]
https://cirqus.swell.store/ Thanks!

**Reply by logeshswell** [reply]
What's the origin-theme version do you use?
Did you try with the latest version?

Also, can you share the screen recording of the issue that you face, as I couldn't follow you properly?
So that I can assist you better.

**Reply by holden** [reply]
https://share.cleanshot.com/rx4dT5JH

**Reply by holden** [reply]
Yes, we're using the latest version of origin 2.0.16. I even cloned a clean version of it just to check it wasn't something we did in our branch. It does exactly the same thing in the latest version of origin.

**Reply by logeshswell** [reply, marked as answer]
Yes, it's a bug at our end. I will keep you posted if I get some updates.

**Reply by logeshswell** [reply]
The temporary workaround would be running `yarn sync`

**Reply by holden** [reply]
Ok, thank you. I will try that.

**Reply by holden** [reply]
Hi Again, 

This only sort of works. 

I now have to add everything into the defaults.json. Are you sure this didn't have anything to do with what I was doing before? Eg I didn't just somehow corrupt my settings messing with the settings on content blocks and could just delete my settings and start over? It's very annoying to not be able to use the editor without it eating the menus and then having to reset them.

**Reply by logeshswell** [reply]
The bug is fixed at our end, can you check and confirm?

**Reply by holden** [reply]
Yes! It appears to be back to normal thank you!
